get-started/codelab/index.md

```diff
@@ -786,7 +786,7 @@ class RandomWordsState extends State<RandomWords> {
 {% endprettify %}
 </li>
 
-<li markdown="1"> Add a `_pushSaved()` function to the RandomWordsState class.
+<li markdown="1"> Add a `_pushSaved()` function to the `RandomWordsState class`.
```

Comment:
On the Flutter website, we don't generally codefont class names. If we did, it would just go on `RandomWordsState`.

Reply:
@Sfshaza So shall I leave it as RandomWordState class or change it to `RandomWordState`? Thanks